### DISCUSSION TITLE
How and where are stocked PDF print in the project ? Problem of slowness

### DISCUSSION THREAD

**Discussion by GlaDal** [opengisch/QField#4007] [Q&A]
Hi, 

I would want to know if it's possible to clean the project folder from a qfieldcloud project ? I have a lot of print PDF generated and I think it's for that reason my project is more and more slow to use after a bit of time and become not usable. 
Could be nice to have an access on the folder on the tablet project or on Qfield cloud ? Maybe it's possible but I don't find it.
Can you help me ?

**Comment by BoswachterMarc** [comment]
Have a look in the qfield cloud documentation: https://docs.qfield.org/get-started/storage/

![afbeelding](https://user-images.githubusercontent.com/57868975/223222941-68a28cf2-0381-4636-acf2-795db4ff3b92.png)

The pdf's are stored in one of those maps.

**Comment by GlaDal** [comment]
Thanks BoswachterMarc for the answer.
This path is only valid for project wich are synchronised with a cable, I think because there are no PDF files in this path. 

To illustrate my need, here is an image of the project folder in Qfield in wich there is one pdf (from print atlas). Imagine that I can have thousands of this kind of PDF file. Reason for my wish to clean this folder for sparing time process when I use this project.

![20230308_161210_temp](https://user-images.githubusercontent.com/7015197/223751535-1b035abc-a957-4789-aa91-880934689754.jpg)

**Comment by GlaDal** [comment]
One question more about PDF document printed from the layout. My PDF doesn't open directly after creation as explained in the image below. For doing that, I need to go to open the project folder then click on the three point at the end of the pdf line and export the PDF in a new folder on my tablet. Then, I open this folder and I can open the PDF. It's not really convenient ! :-) 
Any idea how manage Qfield for opening PDF directly after having printed it from Qfield ?
Thanks for future help !

![image](https://user-images.githubusercontent.com/7015197/223793806-4ecb0ebf-38d8-42ce-8b4c-5cbf61a65785.png)
